apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.tsx

```diff
@@ -1,38 +1,54 @@
 import { useAutoResizer, useFieldValue } from '@contentful/react-apps-toolkit';
 import ChartFooter from 'components/main-app/ChartFooter';
 import ChartHeader from 'components/main-app/ChartHeader';
-import { useEffect, useState } from 'react';
-import { DateRangeType, Row, RunReportResponse } from 'types';
-import DateRange from 'helpers/dateRange.enum';
-import { config } from 'config';
+import { useEffect, useState, useMemo } from 'react';
```

Comment:
I find this name to be confusing because of the term `App`. Why is the entire directory structure called app? 
Doesn't this file and the directory more closely resemble the rendering of the sidebar? 
What about the name `/sidebar/graph/SidebarAnalyticGraph.tsx` or something like that - I'm sure you can come up with something better than I.

Reply:
I think the intention here is that this component can potentially exist not just in the side bar location - I think as we build out more apps we should perhaps build them agnostic to location until it is necessary to define a location for them. Since it is the entire "app" itself that the user sees, I believe that is why this file was named `AnalyticsApp`. What are your thoughts on that?

Reply:
Ahhh I see, I think not calling it sidebar makes sense. I still think the word app is overloaded but it's no big deal. I can commit to this np